pkgs/applications/file-managers/gnome-commander/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, lib
+, fetchurl
+, flex
+, itstool
+, pkg-config
+, libxslt
+, wrapGAppsHook
+, exiv2
+, glib
+, gtk2
+, gvfs
+, libgsf
+, libunique
+, poppler
+, taglib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-commander";
+  version = "1.14.3";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "sha256-eNjc5w+5IrKQnPdneDBTsIESE6TWpJs4dVEM86hO/Xs=";
+  };
+
+  nativeBuildInputs = [
+    flex
+    itstool
+    pkg-config
+    libxslt
+    (wrapGAppsHook.override { gtk3 = gtk2; })
+  ];
+
+  buildInputs = [
+    exiv2
+    glib
+    gtk2
+    libgsf
+    libunique
+    poppler
+    taglib
+  ];
+
+  patches = [ ./gsettings.patch ];
+
+  postPatch = ''
+    substituteInPlace data/org.gnome.${pname}.gschema.xml \
+      --replace /usr/local/share/pixmaps/${pname}/mime-icons "$out"/share/pixmaps/${pname}/mime-icons
+  '';
+
+  postInstall = ''
+    substituteInPlace "$out"/share/applications/org.gnome.${pname}.desktop \
+      --replace Exec=${pname} Exec="$out"/bin/${pname}
+
+    mkdir -p -- "$out"/share/icons/hicolor/scalable/apps
+    ln -s -- "$out"/share/pixmaps/${pname}.svg "$out"/share/icons/hicolor/scalable/apps/${pname}.svg
+  '';
+
```

Comment:
Please enable parallel builds so it does not take aeons to build: https://github.com/NixOS/nixpkgs/pull/93234/files#diff-065ad42d6eda1a9fcca0e33141f7259afb7e4a2d138c588ed47f14d2aa155d68R59